src/storage/txn/scheduler.rs

```diff
@@ -334,6 +350,81 @@ fn process_read(cid: u64, mut cmd: Command, ch: SyncSendCh<Msg>, snapshot: Box<S
                 Err(e) => ProcessResult::Failed { err: e.into() },
             }
         }
+        Command::KeyMvcc { ref ctx, ref key } => {
+            let mut reader = MvccReader::new(snapshot.as_ref(),
+                                             &mut statistics,
+                                             Some(ScanMode::Forward),
+                                             true,
+                                             None,
+                                             ctx.get_isolation_level());
+            let mut ts: u64 = 0;
```

Comment:
The key with small ts should is bigger than key with large ts because of our encoding mode.